antora-ui-camel/src/css/site.css

```diff
@@ -22,4 +22,4 @@
 @import 'release.css';
 @import 'misc.css';
 @import 'community.css';
-@import 'projects.css'
+@import 'projects.css';
```

Comment:
Style changes in this file should be on a separate commit or in a new pull request.